marshmallow/fields.py

```diff
@@ -915,7 +916,6 @@ def _deserialize(self, value, attr, data):
         """Deserialize an ISO8601-formatted time to a :class:`datetime.time` object."""
         if not value:   # falsy values are invalid
             self.fail('invalid')
-            raise err
```

Comment:
Good suggestions, @deckar01 . 

I definitely prefer to have PRs as concise and cohesive as possible. That said, I'm not a stickler on it, and I appreciate when people clean up the code. People are volunteering their valuable time to make these PRs, so I typically don't force people to do too much git "busywork".